mpr-research-data.py

```diff
@@ -7,65 +7,34 @@
 import sqlalchemy as sql
 from google.cloud import storage
 
-logging.basicConfig(
-    format="%(asctime)s %(levelname)-8s [%(filename)s:%(lineno)d] - %(message)s",
-    datefmt="%Y-%m-%dT%H:%M:%S%z",
-    level=logging.INFO
-)
-
-# SETUP CONFIG VARIABLES
-# --------------------------------------------------------------------------
-targetBucketName = os.getenv('GCLOUD_BUCKET', 'mpr-research-data-uploads')
-
-numberOfMonths = os.getenv('NUMBER_OF_MONTHS', 4)
-
-defaultQueryFolder = os.getenv('QUERY_FOLDER', 'queries')
-queryTemplateDict = {'course': os.getenv('COURSE_QEURY', 'courseQuery.sql'),
-                     'retrieve': os.getenv('RETRIEVE_QUERY', 'retrieveQuery.sql')}
-
 
 # FUNCTIONS
 # --------------------------------------------------------------------------
 
-def getDBCreds():
-
-    dbCredsDefaultDict = {'NAME': None,
-                          'USER': None,
-                          'PASSWORD': None,
-                          'HOST': None,
-                          'PORT': 3306}
-    dbCredsDict = {}
+def setUpLogger():
 
-    allKeyPartsFound = True
-    for credPart in dbCredsDefaultDict:
-        dbCredsDict[credPart] = os.getenv(
-            'DB_' + credPart, dbCredsDefaultDict[credPart])
-
-        if not dbCredsDict[credPart]:
-            logging.error(
-                f'Did not find .env variable for M-Write Peer Review production DB key: {credPart}')
-            allKeyPartsFound = False
-
-    if not allKeyPartsFound:
-        sys.exit('Exiting.')
-
-    return dbCredsDict
-
-
-def getGCPCreds() -> dict:
-    gcpCredsDict = json.loads(os.getenv('GCP_KEY'))
-    return gcpCredsDict
+    logWarning = False
+    raiseLogTypeError = None
 
+    try:
+        logLevel = str(os.environ.get('LOG_LEVEL', 'DEBUG')).upper()
```

Comment:
We should use a default level of `INFO`, I think.  That shouldn't be too verbose for production.  For development, we can set the level to `DEBUG` in our own `.env` files.

Reply:
✅